src/main/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -443,16 +443,9 @@ private static void calculatePLIndexToAlleleIndices(final int altAlleles, final
      * @return  number of likelihoods
      */
     private static final int calcNumLikelihoods(final int numAlleles, final int ploidy) {
-        if (numAlleles == 1)
-            return 1;
-        else if (ploidy == 1)
-            return numAlleles;
-        else {
-            int acc =0;
-            for (int k=0; k <= ploidy; k++ )
-                acc += calcNumLikelihoods(numAlleles - 1, ploidy - k);
-            return acc;
-        }
+        //Note: Casting to int instead instead of returning long because values above Integer.MAX_VALUE would not be valid array indices,
+        // and would cause other problems if a PL array needed to be that size
+        return (int)(BinomialCoefficientUtil.binomialCoefficient((numAlleles + ploidy - 1),ploidy));
```

Comment:
I would probably cast this with Math.toIntExact() so we know if we ever overflow.

Reply:
Oh I hadn't thought of that.  Thanks, I'll make that change